### ISSUE TITLE
Using update.sh the dropbear service is shutted down and the ssh connections drops

### ISSUE THREAD

**Issue by rodoviario** [Entware-ng/Entware-ng#24]
Not very sure if it is an issue. When I try to run the update.sh, it shuts down all the old entware services, which is fine, except for dropbear... the ssh conection drops instantly and I am forced to restart the router without knowing if the rest of the update script worked.

**Comment by ryzhovau** [comment]
Can't offer a graceful solution for this scenario, sorry.
Running daemon can block some files while running, so you have another way to upgrade.

**Comment by rodoviario** [comment]
I ended up modifying the script commenting the line that shuts down everything in init.d. I manually shutted down aria, nginx, php-fpm, and the runned the update script.
It came out almost perfect. At first sight everything worked, but looking close, the owncloud client failed to upload files almost always... I started to replace config files with backups. Only when I replaced unslung.rc and func.rc and rebooted, everything started to work properly... before that, it didn's survive a router reset.

I will do a clean installation of entware soon, to leave behind any incompatibility.
Thanks!
